src/main/scala/mesosphere/marathon/api/v2/GroupsResource.scala

```diff
@@ -195,6 +195,24 @@ class GroupsResource @Inject() (
     update("", force, dryRun, body, req, asyncResponse)
   }
 
+  @PATCH
+  def patchRoot(
+    body: Array[Byte],
+    @Context req: HttpServletRequest,
+    @Suspended asyncResponse: AsyncResponse): Unit = sendResponse(asyncResponse) {
+    async {
+      implicit val identity = await(authenticatedAsync(req))
+      val raw = Json.parse(body).as[raml.GroupPartialUpdate]
+      val normalized = GroupNormalization.partialUpdateNormalization(config).normalized(raw)
+
+      val partialUpdate = validateOrThrow(normalized)(Group.validPartialUpdate)
+
+      // TODO: actually update the group in GroupManager without triggering a deployment.
+
+      ok()
+    }
+  }
```

Comment:
Is this roughly how it's supposed to be? Should we add a new method to `GroupManagerImpl`?

Reply:
I think roughly, yes. Let's see how trivial or complex that is. I believe that whole thing is designed to always return a Deployment.